egs/swbd/s5c/local/nnet3/run_cldnn.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# Copyright 2015  Johns Hopkins University (Author: Daniel Povey).
+#           2015  Vijayaditya Peddinti
+#           2015  Xingyu Na
+#           2015  Pegah Ghahrmani
+#           2016  Tom Ko
+# Apache 2.0.
+
+
+# this is a basic lstm script
```

Comment:
change the description for the new script. Describe in detail the architecture used. Described how ivectors are used. Describe the CNN config in detail.